examples/.env.example

```diff
@@ -1,6 +1,7 @@
 # update to match your local environment
 TOKEN_CONTRACT_DIR=~/Aikon/ore-protocol/contracts
 WALLET_PASSWORD=password
+SALT='869C415D 4740001E'
```

Comment:
can you replace this with USER_ACCOUNT_ENCRYPTION_SALT='869...' (and then use something other than '869C415D 4740001E' in the actual env values since it's now in the repo. And use a diff value in prod and staging envs